frontend/src/features/admin-form/responses/FeedbackPage/utils/FeedbackCsvGenerator.ts

```diff
@@ -22,13 +25,27 @@ export class FeedbackCsvGenerator extends CsvGenerator {
       feedback.created && isValid(parseISO(feedback.created))
         ? feedback.created
         : new Date() // If undefined or invalid, use current time
-
     const createdAt = formatInTimeZone(
       feedbackCreatedDate,
       'Asia/Singapore',
       'dd MMM yyyy hh:mm:ss a',
     ) // Format in SG timezone
 
-    this.addLine([createdAt, feedback.comment ?? '', feedback.rating])
+    let feedbackComment = ''
+    if (feedback.comment) {
+      // Check if fieldRecord is a pure number
+      const isPureNumber = PURE_NUMBER_REGEXP.test(feedback.comment)
+      // Check if fieldRecord starts with formula characters
+      const hasFormulaChars = FORMULA_INJECTION_REGEXP.test(feedback.comment)
+      // if fieldRecord is not a pure number, and starts with formula characters,
+      // prefix it with a single quote to prevent formula injection
+      if (!isPureNumber && hasFormulaChars) {
+        feedbackComment = `'${feedback.comment}`
+      } else {
+        feedbackComment = feedback.comment
+      }
```

Comment:
comments still say `fieldRecord` :P 

this logic is identical, maybe can extract to a util `shouldAddQuotes` (or whatever name you desire) and then just call `shouldAddQuotes(feedback.comment)` and `shouldAddQuotes(fieldRecord.getAnswer(colIndex))`

Reply:
good catch + suggestion! made the changes [8ceacfd](https://github.com/opengovsg/FormSG/pull/4950/commits/8ceacfd3e148215efa7a8d4bed55151220c33903)